src/context.ts

```diff
@@ -1,41 +1,25 @@
 import path from "path";
 
-import { EventPayloads, WebhookEvent } from "@octokit/webhooks";
 import merge from "deepmerge";
 
 import type { Logger } from "pino";
 
 import { ProbotOctokit } from "./octokit/probot-octokit";
 import { aliasLog } from "./helpers/alias-log";
 import { DeprecatedLogger } from "./types";
-import { WebhookEvents } from "@octokit/webhooks";
+import type { Repository, Installation, Issue, PullRequest, User } from "@octokit/webhooks-definitions/schema"
+import { EmitterWebhookEventName, BaseWebhookEvent, EmitterWebhookEvent } from "@octokit/webhooks/dist-types/types";
```

Comment:
Could you refresh my mind on why do we need to access `@octokit/webhooks/dist-types`? Is there not a cleaner way to import types here?

Reply:
Is it related to [these](https://github.com/probot/probot/pull/1481/checks?check_run_id=1999551806) PR checks failing?

Reply:
`BaseWebhookEvent` is the equivalent to the old `WebhookEvent` interface from before the types were generated from the schemas.
VSCode just suggested importing them from there since `BaseWebhookEvent` is already imported from there

Reply:
And `EmitterWebhookName1 isn't exported in `@octokit/webhooks` in `index.ts`